docs/readmes/lte/dev_notes.md

```diff
@@ -48,6 +48,72 @@ file is generated by mapping the configuration values from `gateway.mconfig` and
+- The options `Before` and `After` define the order in which services are started and stopped. If a service gets started, the services listed in `Before` and `After` will also be started. If one of the other services gets stopped or its start fails, the service will be stopped. The following dependency graph depicts this order for Magma services.
+
+    ```mermaid
+    graph TD
+        SubscriberDB --> MobilityD
+
+        OVS --> MobilityD
+        MobilityD --> MME
+
+        MobilityD --> PipelineD
+        EnvoyController --> PipelineD
+        EnvoyController --> Envoy
+
+        OVS --> PipelineD
+        PipelineD --> MME
+
+        PipelineD --> ConnectionD
+        PipelineD --> LiAgent
+
+        OVS --> ConnectionD
+        ConnectionD --> MME
+
+        SctpD --> MME
+        SessionD --> MME
+    ```
+
+- The options [Wants](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=) and [Requires](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=) specify a set of services that will be started if the configuring service starts. If the listed services fail to start, this has no impact on the service start in the case of `Wants`. Considering `Requires`, the service will not be started if the option `After` is set on the failing service.
+The subsequent figure illustrates both configuration options for Magma services. For example, `EnvoyController → Envoy` is read as "If EnvoyController starts, require to start Envoy as well." and `MobilityD → SubscriberDB` as "If MobilityD starts, a start of SubscriberDB is wanted.".
+
+    ```mermaid
+        graph TD
+        MME --> SctpD
+
+        MobilityD -.-> OVS
+        MobilityD -.-> SubscriberDB
+
+        PipelineD -.-> OVS
+        EnvoryController --> Envoy
+    ```
+
+    (Wants = Doted line, Requires = Normal line)
+
+- The option [PartOf](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#PartOf=) includes a set of other services restarted or stopped if the service is restarted or stopped. The following graph shows the `PartOf` configuration for Magma services. For example, `ConnectionD → MME` can be read as "If ConnectionD restarts, then an MME restart will also initiate."
+
+    ```mermaid
+    graph TD
+        LiAgent --> PipelineD
+        Envoy --> PipelineD
+        EnvoyController --> PipelineD
+
+        MME --> SessionD
+        MME --> PipelineD
+        MME --> MobilityD
+
+        SessionD --> MME 
+        PipelineD --> MME
+        MobilityD --> MME 
+        ConnectionD --> MME
+
+        PipelineD --> MobilityD
+        PipelineD --> OVS
+    ```
+
+   ➡️ The systemd configuration options `Wants`, `Requires`, and `PartOf` only define which services have to be started, restarted or stopped. The order of starts/restarts/stops is specified by `Before` and `After`.
```

Comment:
_Optional:_ Instead of the icon an [banner](https://docusaurus.io/docs/markdown-features/admonitions) might be really nice here.

Reply:
Is not supported in the used Docusaurus version.